### DISCUSSION TITLE
minio change default storage path

### DISCUSSION THREAD

**Discussion by clarkstyx** [minio/minio#15623] [Q&A] [answered]
so minio server was installed by default on its default path or the main partition where all the other containter resides.. its current task is to store video and images feed however its becoming full and other services stopped because of this. So I want to know what my options are, 1. Is there any command to change the default bucket path to a different disk / path? 2. Should I just make another instance of MINIO server on a different partition / volume so all the buckets will go directly there? Please be kind to me, Im not a programmer, I was just given a challenge by my boss to fix it. Thanks in advance.

**Comment by ravindk89** [comment, marked as answer]
We need a lot more information here to be helpful.

> so minio server was installed by default on its default path or the main partition where all the other containter resides.

I'm not sure how to interpret this - what operating system is this on, and how was it installed? By "container" do you mean that you are running MinIO as an image?

> its current task is to store video and images feed however its becoming full and other services stopped because of this.

It sounds like MinIO is using the root drive or some other storage resource that is required by other services, and things are crawling to a halt.

> Is there any command to change the default bucket path to a different disk / path?

MinIO has no "default" bucket path. When you start up minIO, you specify the path to use. 

> Should I just make another instance of MINIO server on a different partition / volume so all the buckets will go directly there?

That's part of the potential solution here. You need a dedicated storage volume on which you can run a separate MinIO process, then use something like `mc mv --recursive` to move each bucket one-by-one from the old volume to the 'new' volume.

It's important to know - are you accessing these videos only through MinIO, or are you also accessing these files through a network mount or something similar? We made a significant change in latest stable with how MinIO handles files on disk for single-drive setups that might impact your usage.
